builtin/game/item_entity.lua

```diff
@@ -36,6 +36,9 @@ core.register_entity(":__builtin:item", {
 	itemstring = '',
 	physical_state = true,
 	age = 0,
+	-- The items are mergeable as long as this property is not false,
+	-- so we use nil to save a few bytes unless it will be set.
+	mergeable = nil,
```

Comment:
@sapier I don't get were you see any problem here. By explicit checking for ```true``` its made sure to behave like currently.

@RobertZenz IMO you can stay with ```mergable```


And that this pull is still open is a **shame** for minetest. Not that it has been forgotten, i requested a second approve like what, 5 times?